### ISSUE TITLE
Build error, srdfdom

### ISSUE THREAD

**Issue by lorepieri8** [ros-planning/moveit2#458]
### Description

Cannot build successfully.

### Your environment
* ROS Distro: Foxy
* OS Version: e.g. Ubuntu 20.04
* Source  build
* 
### Steps to reproduce
Follow build instructions (https://moveit.ros.org/install-moveit2/source/) and type:
`colcon build --event-handlers desktop_notification- status- --cmake-args -DCMAKE_BUILD_TYPE=Release`

### Backtrace or Console output
https://gist.github.com/lorepieri8/9188c0e7e0a094374e10a6b52379d26f

### Proposed Solution:
sudo apt-get install ros-foxy-srdfdom

**Comment by peterdavidfagan** [comment]
@lorepieri8 are you still facing this issue? 

I recently was able to build MoveIt 2 from source using the instructions you referenced (https://moveit.ros.org/install-moveit2/source/). My environment setup matches the description you provided above. 

I did however encounter a similar error relating to target "urdfdom_headers::urdfdom_headers" not being found. This initially occurred when starting from the following instructions (http://moveit2_tutorials.picknik.ai/doc/getting_started/getting_started.html).

I did cross reference both sets of installation instructions, one notable difference is that the former installs python3-rosdep while the latter installs python3-rosdep2. 

I  haven't fully gotten to the bottom of this installation issue yet but I am happy to debug further, if you are still encountering this issue.
